### ISSUE TITLE
Make the blocks-layout system more flexible

### ISSUE THREAD

**Issue by beligum** [republic-of-reinvention/com.stralo.framework#79]
Now, the <blocks-layout> tag is closely integrated with to the Bootstrap container class.
This means there's no way to "disable" certain parts of the page content inside that container.
We want to limit the edit/layout-ability of the blocks to the places that are wrapped by a <blocks-layout> tag.

**Comment by beligum** [comment]
Note that the original `<blocks-layout>` had this default content:
```
<template>
    <div data-property="container" class="container">
        <div class="row">
            <div class="col-md-6">
                <blocks-text></blocks-text>
            </div>
            <div class="col-md-6">
                <blocks-text></blocks-text>
            </div>
        </div>
    </div>
</template>
```

But since we can now use it like this:
```
<div class="container">
    <div class="row">
        <div class="col-md-12">
            <blocks-breadcrumb></blocks-breadcrumb>
        </div>
    </div>
    <div class="row">
        <div class="col-md-2">
            <blocks-spacer />
        </div>
        <div class="col-md-6">
            <blocks-layout data-property="facts"></blocks-layout>
        </div>
        <div class="col-md-4">
            <blocks-layout data-property="media"></blocks-layout>
        </div>
    </div>
</div>
```

it doesn't really make sense anymore to include the container in the block, because we buy a lot more flexibility when the container is outside of the layout block. However, we can't wipe the `data-property="container"` attribute, since it's our hook to save everything inside the layout block and we can't change its name because of backwards-compatibility.

Bottom line, let's change the default content of a layout block to:

```
<template>
    <div data-property="container">
        <blocks-text></blocks-text>
    </div>
</template>
```

**Comment by beligum** [comment]
Hmmm, it's more complicated than that, because all previously saved pages already have the `class="container"` attribute stored in the database. On top of that, by changing the default content of the block, we basically indicate it should be embedded in a wrapping container by default, which is not what we want, because pulling the container out of the `<blocks-layout>` effectively disables it's layout engine.

Pulling it out is only realistic if we want to limit the layout capabilities of a page (eg. see the stralo-page-resource template), so we decided to keep it the way it is, and override the default content when we need to (like in stralo-page-resource)
